packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -6092,7 +6111,14 @@ export default class Meeting extends StatelessWebexPlugin {
       data: {trigger: trigger.USER_INTERACTION, canProceed: false},
     });
     const leaveReason = options.reason || MEETING_REMOVED_REASON.CLIENT_LEAVE_REQUEST;
-
+    // add
+    if (leaveReason === MEETING_REMOVED_REASON.CLIENT_LEAVE_REQUEST_TAB_CLOSED) {
```

Comment:
we're not sending any leaveReason at the moment in Cantina, so we need to fix it there too

Reply:
added new judgement for isUnverifiedGuest, please help to check again.